Kinvey.Core/Core/AbstractKinveyClientRequest.cs

```diff
@@ -528,7 +587,9 @@ public virtual T Execute()
             }
             try
             {
-				return JsonConvert.DeserializeObject<T>(response.Content);
+                var task = response.Content.ReadAsStringAsync();
+                task.Wait();
```

Comment:
Is this going to cause blocking?

Reply:
That's fine. As we refactor, we may find that we no longer need the `Execute` method. If we find that is the case, we can remove it.